composer.lock

```diff
@@ -10012,5 +10016,5 @@
     "platform-overrides": {
         "php": "7.1.3"
     },
-    "plugin-api-version": "2.0.0"
```

Comment:
I dont know if this is OK 🤔  @PierreRambaud what did we say about `plugin-api-version` ?

Reply:
@matks it's because I'm running on v1 locally I think I'll revert it manually?